test/MultiAssembly.ByAssembly.docsproj

```diff
@@ -34,4 +38,10 @@
     <Assembly Include="..\src\Mintlify.Core\bin\$(Configuration)\net8.0\Mintlify.Core.dll" />
   </ItemGroup>
 
+  <PropertyGroup>
+    <_DotNetDocsTasksFolder>$(MSBuildThisFileDirectory)..\src\CloudNimble.DotNetDocs.Sdk\bin\Debug\</_DotNetDocsTasksFolder>
```

Comment:
This is the only slightly-weird thing. Since you're loading the props and targets from the repo and not from an SDK package, it becomes necessary to tell your SDK targets where the matching tasks implementations are.